modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase

```diff
@@ -276,12 +276,104 @@ definition {
 	}
 
 	@description = "LPS-88573 - Verify that once an order option is made by the user, this definition should be saved by the system during the user stay in the Metrics context, even when the user makes a search query, reloads the page"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test SavedOrderOptionStaysInMetricsContext {
+		Workflow.duplicateSingleApproverWorkflow(workflowName = "Single Approver A");
+
+		Workflow.duplicateSingleApproverWorkflow(workflowName = "Single Approver B");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
+			entrySubtitle = "Subtitle",
+			entryTitle = "Blogs Entry A");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
```

Comment:
For this Blogs macro could you please change it to 
`JSONBlog.addEntry(
				entryContent = "Blogs Entry Content",
				entryTitle = "Blogs Entry Title");`